src/module.c

```diff
@@ -3314,8 +3378,8 @@ int RM_StreamDelete(RedisModuleKey *key, RedisModuleStreamID *id) {
  *             //
  *             // ... Do stuff ...
  *             //
- *             RedisModule_Free(field);
- *             RedisModule_Free(value);
+ *             RedisModule_FreeString(ctx, field);
+ *             RedisModule_FreeString(ctx, value);
```

Comment:
Looks like an important but perhaps different commit?

Reply:
Well, yes, I realized after the streams PR was merged... Do you want me to do it in a different PR?

RM_Free() works actually, but not when combined with auto memory.

Reply:
This is now done in unstable via #8757.